packages/suite-base/src/providers/CurrentLayoutProvider/index.test.tsx

```diff
@@ -56,8 +56,8 @@ function makeMockLayoutManager() {
     off: jest.fn(/*noop*/),
     setError: jest.fn(/*noop*/),
     setOnline: jest.fn(/*noop*/),
-    getLayouts: jest.fn().mockImplementation(mockThrow("getLayouts")),
```

Comment:
I don't get the reason why this mock and all the tests related to it had to be removed.

Reply:
Because in this test (CurrentLayoutProvider), it should not be using getLayouts, so the mock would throw an error in case of being used. Now we add the functionality of "layoutManager.getLayouts" and we don't want to throw an error in this case. But the "getLayout" is not used, so I will return with that mockThrow.